dev3/2024-07-02/example-metadata/rembi-modules.json

```diff
@@ -0,0 +1,115 @@
+{
+    "@context": [
+        "https://w3id.org/ro/crate/1.1/context",
+        {
+            "organism_classification": "https://schema.org/taxonomicRange",
+            "BioChemEntity": "https://schema.org/BioChemEntity",
+            "channel": "https://w3id.org/reproduceme#Channel",
```

Comment:
Happy to use this for the challenge, but this would be one thing I would look to update beyond.

Reply:
For my culture, why not point directly to https://www.openmicroscopy.org/Schemas/Documentation/Generated/OME-2016-06/ome_xsd.html#Channel here?